Document how to get the API Server to work

Apparently this plugin needs an "API Server".

How to install that API Server on a ARM-powered KODI "TV box"?

It would be nice if it would at least give some hints where to get that "API Server" from. Without one, the plugin seems not to be usable.

Is https://github.com/lbryio/lbrytv the one?

Downloaded and built it with go, but then?

```
FreeBSD% /home/liveuser/go/bin/lbrytv 
{"buildCommit":"unknown","buildDate":"unknown","buildVersion":"unknown","level":"info","logLevel":"info","mode":"production","msg":"standard logger configured"}
{"level":"info","module":"monitor","msg":"sentry disabled (no DSN configured)"}
{"dsn":"postgres://lbrytv:lbrytv@localhost/lbrytv?sslmode=disable","level":"info","module":"storage","msg":"connecting to the DB"}
{"dsn":"postgres://lbrytv:lbrytv@localhost/lbrytv?sslmode=disable","level":"info","module":"storage","msg":"DB connection failed"}
panic: dial tcp [::1]:5432: connect: connection refused

goroutine 1 [running]:
main.main()
        /home/liveuser/lbrytv/main.go:41 +0x766
FreeBSD% /home/liveuser/go/bin/collector 
{"buildCommit":"unknown","buildDate":"unknown","buildVersion":"unknown","level":"info","logLevel":"info","mode":"production","msg":"standard logger configured"}
{"dsn":"postgres://collector:ZnPfJ!4vTb-nURv8M!JkkuAcjYLYbkiX@localhost:5433/collector?sslmode=disable","level":"info","module":"storage","msg":"connecting to the DB"}
{"dsn":"postgres://collector:ZnPfJ!4vTb-nURv8M!JkkuAcjYLYbkiX@localhost:5433/collector?sslmode=disable","level":"info","module":"storage","msg":"DB connection failed"}
panic: dial tcp [::1]:5433: connect: connection refused

goroutine 1 [running]:
github.com/lbryio/lbrytv/apps/environment.ForCollector(0xc74940)
        /home/liveuser/lbrytv/apps/environment/apps.go:22 +0x3b0
github.com/lbryio/lbrytv/apps/collector/cmd.glob..func3(0x1170d20, 0x11ac458, 0x0, 0x0)
        /home/liveuser/lbrytv/apps/collector/cmd/serve.go:19 +0x26
github.com/spf13/cobra.(*Command).execute(0x1170d20, 0xc000032200, 0x0, 0x0, 0x1170d20, 0xc000032200)
        /home/liveuser/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:846 +0x2c2
github.com/spf13/cobra.(*Command).ExecuteC(0x1170d20, 0x8bcc83b7490, 0x476aef, 0xc000110000)
        /home/liveuser/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:950 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        /home/liveuser/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:887
github.com/lbryio/lbrytv/apps/collector/cmd.Execute()
        /home/liveuser/lbrytv/apps/collector/cmd/serve.go:32 +0x31
main.main()
        /home/liveuser/lbrytv/apps/collector/main.go:16 +0x8a
```

--- a/resources/settings.xml
+++ b/resources/settings.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8" standalone="yes"?>
 <settings>
   <category label="30000">
-    <setting id="lbry_api_url" type="text" option="urlencoded" label="30005" default="http://localhost:5279"/>
+    <setting id="lbry_api_url" type="text" option="urlencoded" label="30005" default="https://api.lbry.tv/api/v1/proxy"/>
     <setting id="nsfw" type="bool" label="30001" default="false"/>
     <setting id="items_per_page" type="number" label="30002" default="20"/>
     <setting type="lsep" label="30003"/>

